packages/core/src/build/service.ts

```diff
@@ -128,7 +130,8 @@ export class BuildService extends Emittery<BuildServiceEvents> {
       const schema = await buildSchema({
         schemaFile: this.common.options.schemaFile,
       });
-      this.emit("newSchema", { schema });
+      const graphqlSchema = buildGqlSchema(schema);
```

Comment:
I think it would be clearer if this happens elsewhere (ServerService, maybe?). The BuildService should only be responsible for building & validating the new "app schema". Constructing the GraphQL schema from the "app schema" should be downstream.

Reply:
Hmmm it kinda makes sense to me to keep if here but open to change it. This is also the way it is done currently